httpretty/core.py

```diff
@@ -937,6 +957,53 @@ def match_uriinfo(cls, info):
         return (None, [])
 
     @classmethod
+    def match_https_hostname(cls, hostname):
+        items = sorted(
+            cls._entries.items(),
+            key=lambda matcher_entries: matcher_entries[0].priority,
+            reverse=True,
+        )
+        for matcher, value in items:
+            if matcher.info is None:
+                pattern_with_port = "https://{}:".format(hostname)
+                pattern_without_port = "https://{}/".format(hostname)
```

Comment:
These `{}` are unnecessarily Python 2.6-incompatible; it's pretty painless to just do `{0}` instead, and then it works with 2.6.